pp.c

```diff
@@ -7700,6 +7700,23 @@ PP(pp_anonconst)
 }
 
 
+/* Helper function for use during signature argument handling from @_.
+ * Replaces elements in an AV with a new SV cloned from the original value
+ * at each position from startix onwards until endix.
+ */
+#define av_refresh_elements_range(av, startix, endix)  S_av_refresh_elements_range(aTHX_ av, startix, endix)
+STATIC void
+S_av_refresh_elements_range(pTHX_ AV *av, IV startix, IV endix)
+{
+    for(IV ix = startix; ix < endix; ix++) {
+        SV **svp = av_fetch(av, ix, FALSE);
```

Comment:
Unless var av is reachable from PP lang, use fetch_simple() variants, we dont need to call `TIEARRAY` PP methods here.